OnboardPietroPieri/App.tsx

```diff
@@ -73,7 +94,11 @@ const App = () => {
 
       <TextField title="Senha" type="password" onChange={setPassword} />
 
-      <LoginButton body="Entrar" onPress={validateLogin} />
+      <LoginButton
+        body={loading ? "Loading..." : "Entrar"}
```

Comment:
Nit: do ponto de vista do usuário, é importante o idioma estar consistente
```suggestion
        body={loading ? "Carregando..." : "Entrar"}
```